src/main/kotlin/no/nav/familie/ef/sak/mapper/SøknadsskjemaMapper.kt

```diff
@@ -279,13 +320,16 @@ object SøknadsskjemaMapper {
             list?.map {
                 Arbeidsgiver(arbeidsgivernavn = it.arbeidsgivernavn.verdi,
                              arbeidsmengde = it.arbeidsmengde?.verdi,
-                             fastEllerMidlertidig = it.fastEllerMidlertidig.verdi,
+                             fastEllerMidlertidig = EnumTekstverdiMedSvarId(it.fastEllerMidlertidig.verdi,
+                                                                            it.fastEllerMidlertidig.svarId ?: MANGLER_SVAR_ID),
                              harSluttdato = it.harSluttdato?.verdi,
                              sluttdato = it.sluttdato?.verdi)
             }?.toSet() ?: emptySet()
 
     private fun tilDomene(situasjon: KontraktSituasjon): Situasjon =
-            Situasjon(gjelderDetteDeg = StringUtils.join(situasjon.gjelderDetteDeg.verdi, "; "),
+            Situasjon(gjelderDetteDeg = EnumTekstverdiMedSvarId(StringUtils.join(situasjon.gjelderDetteDeg.verdi,
+                                                                                 ";"), // TODO: whitespace etter ;?
+                                                                StringUtils.join(situasjon.gjelderDetteDeg.svarId, ";")),
```

Comment:
Denne synes jeg kan bli vrien å "jobbe med"  - for å få ut verdiene her må vi isåfall splitte på `;` for å vite hvilke valg brukeren har gjort... 🤔 Kan kanskje stå sånn inntil videre så ser vi hva vi trenger i stønadsvilkårene?

Reply:
Ja, ser det ikke blir optimalt. Kanskje `svarId` feltet i `EnumTekstverdiMedSvarId` bør være en generisk type?